bin/test_stack

```diff
@@ -79,10 +79,11 @@ sudo ip link set t1sw2-eth28 up
 sleep 50.3231 &
 sleep 32
 test_forch -healthy
-test_varz_value process_state{process=\"faucet\"} forch
-test_varz_value process_state{process=\"gauge\"} forch
-test_varz_value process_state{process=\"forch\"} forch
-test_varz_value process_state{process=\"sleep\"} forch
+test_varz_value process_state{process=\"faucet\"} 8302
+test_varz_value process_state{process=\"gauge\"} 8080 forch
+test_varz_value process_state{process=\"forch\"} 8080 forch
+test_varz_value process_state{process=\"sleep\"} 8080 forch
+test_varz_value process_state{process=\"sleep\"} 8080 random_string
```

Comment:
https://xkcd.com/221/ -- not really random :-) -- better use something like unknown or error -- the intention behind it

Reply:
Addressed